qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/engineering_diffraction_io.py

```diff
@@ -35,11 +35,10 @@ def encode(self, obj, _=None):  # what obj = EngineeringDiffractionGui
             obj_dic["settings_dict"] = presenter.settings_presenter.settings
         else:
             obj_dic["settings_dict"] = presenter.settings_presenter.model.get_settings_dict(SETTINGS_KEYS_TYPES)
-        if data_widget.presenter.get_loaded_workspaces():
-            obj_dic["data_loaded_workspaces"] = [*data_widget.presenter.get_loaded_workspaces().keys()]
+        if data_widget.model._data_workspaces.get_ws_names_dict():
+            obj_dic["data_workspaces"] = data_widget.model._data_workspaces.get_ws_names_dict()
             obj_dic["fit_results"] = data_widget.model.get_fit_results()
             obj_dic["plotted_workspaces"] = [*data_widget.presenter.plotted]
-            obj_dic["background_params"] = data_widget.model.get_bg_params()
```

Comment:
Why is it fine to remove this background_params entry, also on line 72 of this file?